src/olympia/amo/tests/__init__.py

```diff
@@ -750,7 +751,7 @@ class ESTestCase(TestCase):
     # ES is slow to set up so this uses class setup/teardown. That happens
     # outside Django transactions so be careful to clean up afterwards.
     mock_es = False
-    exempt_from_fixture_bundling = True  # ES doesn't support bundling (yet?)
+    indexes_names = None  # Changed below.
```

Comment:
imho index_names but I'm no native too :D
 
 
On Thu, Mar 24, 2016, at 02:35 PM, Mathieu Pillard wrote:
> In src/olympia/amo/tests/__init__.py[1]:


> > @@ -750,7 +751,7 @@ class ESTestCase(TestCase): >      # ES is slow
> > to set up so this uses class setup/teardown. That happens >      #
> > outside Django transactions so be careful to clean up afterwards. >
> > mock_es = False > -    exempt_from_fixture_bundling = True  # ES
> > doesn't support bundling (yet?) > +    indexes_names = None  #
> > Changed below.
>
> Or indice_names ? indices_names ? English is complicated :)
> —
> You are receiving this because you were assigned.
> Reply to this email directly or view it on GitHub[2]
>
 

Links:

  1. https://github.com/mozilla/addons-server/pull/1961#discussion_r57316351
  2. https://github.com/mozilla/addons-server/pull/1961/files/137f28656fe017119718dc6666c6b0846a35df7a#r57316351